### PR TITLE
Add "show" and "unpin" options to planex-pin

### PR DESCRIPTION
**Pull request by TimSmithCtx**
Quality-of-life improvements for planex-pin. To show the current state
of a pin, use "planex-pin --show <package>", and to remove a pin, use
"planex-pin --unpin <package>".

This removes one need for the user to know the implementation of pins.

### CODE REVIEW

File: planex/cmd/pin.py [hunk 1]
```diff
@@ -139,15 +147,32 @@ def main(argv=None):
...
+            if e.errno not in (errno.ENOENT,):
+                raise
+            print("Package '{}' is not pinned".format(package_name))
+            pass
+
+    if args.unpin:
+        try:
+            os.remove(default_output)
+        except:
+            pass
```
Comment (mseri) [inline planex/cmd/pin.py]: I think it would be better to fail ungracefully if the removal goes wrong

Comment (TimSmithCtx) [reply planex/cmd/pin.py]: For any other reason than ENOENT, maybe. Unpinning a thing which is already not pinned should be a no-op.

Comment (srowe) [reply planex/cmd/pin.py]: Why not test for the existence of the file first? This isn't a command where we need to worry much about concurrency.

Comment (TimSmithCtx) [reply planex/cmd/pin.py]: It was easier to just catch the error, given that I have some code doing just that right above it to borrow.

**Comment by srowe** [comment]
Can you add a SoB?

**Pull request merged**